sktime/forecasting/online_ensemble.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pandas as pd
+from sktime.forecasting.base._base import DEFAULT_ALPHA
+from sktime.forecasting.compose._ensemble import EnsembleForecaster
+from sktime.forecasting.model_selection import OnlineSplitter
+from .ensemble_algorithms import EnsembleAlgorithms
+
+
+class OnlineEnsembleForecaster(EnsembleForecaster):
+    """Online Updating Ensemble of forecasters
+
+    Parameters
+    ----------
+    ensemble_algorithm : ensemble algorithm
+    forecasters : list of (str, estimator) tuples
+    n_jobs : int or None, optional (default=None)
+        The number of jobs to run in parallel for fit. None means 1 unless
+        in a joblib.parallel_backend context.
+        -1 means using all processors.
+    """
+
+    _required_parameters = ["forecasters"]
+
+    def __init__(self, forecasters, ensemble_algorithm=None, n_jobs=None):
+        self.n_jobs = n_jobs
+        if ensemble_algorithm is None:
+            self.ensemble_algorithm = EnsembleAlgorithms(len(forecasters))
+        else:
+            self.ensemble_algorithm = ensemble_algorithm
```

Comment:
This should probably happen in fit to comply with scikit-learn's API and to make sure `set_params` works correctly

Reply:
I guess I was just trying to set the default ensemble algorithm as a parameter that depends on the length of forecasters and chose to do it in the init. If I just did `self.ensemble_algorithm = ensemble_algorithm` and then added
`if ensemble_algorithm is None:
            self.ensemble_algorithm = EnsembleAlgorithms(len(forecasters))`
In the fit function, is this what you are thinking about?

Reply:
Just checked, I think `set_params` is working correctly for the `ensemble_algorithm` parameter.